packages/analytics-js-integrations/src/integrations/Criteo/utils.js

```diff
@@ -30,6 +31,15 @@ const handleCommonFields = (rudderElement, hashMethod) => {
     setEmail.hash_method = hashMethod;
     setEmail.email = hashMethod === 'md5' ? md5(email) : email;
     finalRequest.push(setEmail);
+
+    // When using email hashes, send sha256 in addition to MD5
+    if(hashMethod === 'md5' || hashMethod === 'sha256') {
```

Comment:
if we need to support sha256 also then the above logic for handling md5 hashing also needs to be refactored, we will check and come back. meanwhile, please share the documentation link for this change.

Reply:
You can find documentation here, it shows a tag implementation with both sha256 and md5 emails: https://help.criteo.com/kb/guide/en/intro-to-the-criteo-onetag-8fjCDwCENw/Steps/775595,853887,2616497,2616569,2616738,2617588

My PR proposes a naïve and trivial implementation to get the discussion started, I'll be happy to rework it as per your recommendations or let you handle the rest as you'll find suitable.

Reply:
We will look into it and let you know.